src/CardContent/FileDetails/FileDetails.types.ts

```diff
@@ -9,6 +9,14 @@ export type FileDetailsProps = {
    * The download button text on search.
    */
   downloadButtonTextOnSearch?: string;
+  /**
+   * The download href link.
+   */
+  downloadLinkHref?: string;
+  /**
+   * The download href link target attribute.
+   */
+  downloadLinkTarget?: string;
```

Comment:
I don't think string is the correct type here. It's actually a string union. Seems like react has a `HTMLAttributeAnchorTarget` type somewhere we should presumably be able to access.

![image](https://github.com/IPG-Automotive-UK/react-ui/assets/27866636/0f7ccfa5-f485-4aa6-9cb3-29755e8458fa)